blockchain/types/accountkey/account_key_legacy.go

```diff
@@ -45,6 +45,11 @@ func (a *AccountKeyLegacy) IsCompositeType() bool {
 	return false
 }
 
+// AccountKeyLegacy cannot check the public key, just return address of the public key argument always
+func (a *AccountKeyLegacy) IsContainedKey(recoveredKey *ecdsa.PublicKey) bool {
+	return true
+}
```

Comment:
1. Unlike the comment, it just returns a bool.
2. It's awkward for ContainsPubkey(pub) is true for any pubkey. I think it should be false

Reply:
1. Thanks, It should be fixed to return boolean, not address.
2. Right, it's weird a bit But It should return `true` so that the user can check it. If we return false and then `nil address` in the recoverMessage, the user don't have a chance to check.
How do you think the following comments?
```go
// AccountKeyLegacy cannot check the public key, just return true always so that the user can verify it
func (a *AccountKeyLegacy) IsContainedPubkey(recoveredKey *ecdsa.PublicKey) bool {
	return true
}
```